### ISSUE TITLE
Interact with "spending cap" metamask window

### ISSUE THREAD

**Issue by s1m0nDevs** [ChainSafe/dappeteer#309]
What about the spending cap window? Trying to access it from `metamask.page`, but once I did `setupMetaMask` and tried to get the current metamask popup window from getMetaMaskWindow I am getting the general metamask page with the assets, but not the actual one with the current transaction.
![image](https://user-images.githubusercontent.com/53870756/233055686-6cf4f86d-5dab-4583-8741-1f512b534600.png)

**Comment by s1m0nDevs** [comment]
Yep, for example on [sushiswap](https://www.sushi.com/swap?fromChainId=137&fromCurrency=0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174&toChainId=137&toCurrency=NATIVE) or 1inch if you wanna swap usdt/usdc(for example).

**Comment by BeroBurny** [comment]
thanks for pointing out, issue moved to feature

**Comment by Julink-eth** [comment]
@s1m0nDevs 
Hello! Have you been able to interact with it ?
I'm trying but no success for now.
